test/kubernetes-test.sh

```diff
@@ -6,7 +6,7 @@
 # once a new label is added to a pod. If we delay the identity change
 # process the test will fail.
 
-helm template install/kubernetes/cilium \
+helm install cilium install/kubernetes/cilium \
```

Comment:
I assume the changes in this file is to overcome the issue, which _--kube-version_ flag is not available in helm 3 template CLI. Nice :tada: 

One minor nit, the current script is kind of idempotent (e.g. you can re-run a few times for same cluster), however _helm install_ is not. If you want, you can make it to `helm upgrade -i` (I used this command in CI a lot for same use case)

> helm upgrade -h
> ...
>   -i, --install                      if a release by this name doesn't already exist, run an install

Reply:
Hey @sayboras 
Yeah, this is to overcome the default faked value provided for Built-In objects like `Capabilities.KubeVersion` when generating templates. I was just testing it out though. I will see the use of the command you mentioned. But, it seems to me that this might not be fixed by just doing a `helm install` or `helm upgrade` here. In the ginkgo test suite, we redeploy cilium multiple times, each time it makes use of this template generated earlier using the `helm template` command. See [here](https://github.com/cilium/cilium/blob/master/test/helpers/kubectl.go#L2316-L2345)
I have already asked a question related to this on slack and I will replace the use of generated templates for deploying cilium with running `helm upgrade -i` if everyone agrees.